teleop_twist_keyboard.py

```diff
@@ -81,6 +79,8 @@ def vels(speed,turn):
 	pub = rospy.Publisher('cmd_vel', Twist, queue_size = 1)
 	rospy.init_node('teleop_twist_keyboard')
 
+        speed = rospy.get_param("~speed", 0.5)
+        turn = rospy.get_param("~turn", 1.0)
```

Comment:
Indentation here doesn't match the rest of this function :(

Aside from that, this is awesome!